### PR TITLE
fix extra counts slots

### PR DESCRIPTION
**Pull request by ogibson-biomage**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-1448

#### Link to staging deployment URL 

https://ui-oliver-p166.scp-staging.biomage.net/

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging
- [ ] Passed integration tests 

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [X ] Photo of a cute animal attached to this PR
image

### CODE REVIEW

**Review by aerlaut — APPROVED** [review]
LGTM

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -231,7 +231,7 @@ jobs:
             KUBERNETES_ENV="staging"
             REPLICA_COUNT="2"
             IMAGE_GLOB="${IMAGE_TAG/$GITHUB_SHA/*}"
-            MEMORY_REQUEST="4Gi"
+            MEMORY_REQUEST="8Gi"
```
Comment (aerlaut) [inline .github/workflows/ci.yaml:234]: Don't forget to revert this

File: pipeline-runner/R/gem2s-2-load_cellranger_files.R [hunk 2]
```diff
@@ -50,6 +50,11 @@ call_read10x <- function(config) {
     message("Reading files from ", sample_dir, ' --> ', paste(sample_fpaths, collapse = ' - '))
 
     counts <- Seurat::Read10X(sample_dir, gene.column = 1)
+
+    if(length(names(counts))){
+      counts <- do.call(rbind,counts)
+    }
```
Comment (alexvpickering) [inline pipeline-runner/R/gem2s-2-load_cellranger_files.R:56]:
```suggestion
    if(is(counts, 'list')) {
      slot <- 'Gene Expression'
      # questionable: grab first slot if no gene expression
      if (!slot %in% names(counts)) slot <- names(counts)[1]
      counts <- counts[[slot]]
    }
```

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ogibson-biomage) [reply .github/workflows/ci.yaml:234]:
```suggestion
            MEMORY_REQUEST="4Gi"
```

**Pull request merged**